src/libraries/Microsoft.Extensions.DependencyInjection.Abstractions/src/ActivatorUtilities.cs

```diff
@@ -361,10 +434,10 @@ public FactoryParameterContext(Type parameterType, bool hasDefaultValue, object?
             out int?[] matchingParameterMap)
         {
             ConstructorInfo? constructorInfo = null;
-            int?[]? parameterMap = null;
+            int?[]? parameterMap;
```

Comment:
Why change `parameterMap` to an `out`, but not `constructorInfo`? It seems they should use the same pattern.

Reply:
Done. thanks